pkg/cmd/extension/manager.go

```diff
@@ -519,15 +525,30 @@ func (m *Manager) upgradeExtensions(exts []Extension, force bool) error {
 			fmt.Fprintf(m.io.Out, "%s\n", err)
 			continue
 		}
-		fmt.Fprintf(m.io.Out, "upgrade complete\n")
+		currentVersion := f.currentVersion
+		latestVersion := f.latestVersion
+		if !f.IsBinary() && len(currentVersion) > 8 {
+			currentVersion = currentVersion[:8]
+		}
+		if !f.IsBinary() && len(latestVersion) > 8 {
+			latestVersion = latestVersion[:8]
+		}
+		if dryRun {
+			fmt.Fprintf(m.io.Out, "would have upgraded from %s to %s\n", currentVersion, latestVersion)
+		} else {
+			fmt.Fprintf(m.io.Out, "upgraded from %s to %s\n", currentVersion, latestVersion)
```

Comment:
This is a small change in existing behavior, I feel like it is nicer to print out the actual version change.